src/main/resources/application.properties

```diff
@@ -1 +1 @@
-
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
let's enable the console so that we could have access to our db from the browser